lib/datadog/tracing/contrib/lograge/instrumentation.rb

```diff
@@ -33,7 +33,7 @@ def custom_options(event)
                   service: correlation.service.to_s,
                   version: correlation.version.to_s
                 },
-                ddsource: ['ruby']
+                ddsource: Logging::Ext::DD_SOURCE
               }
```

Comment:
I guess maybe not for this PR, but /me spots something that looks like it should be inside a `Correlation#to_log_hash_format` and not repeated twice :)

Reply:
I agree, it wasn't super clean to do it in this PR, so I didn't refactor that part.